cz.zeropage.Formiko.json

```diff
@@ -0,0 +1,300 @@
+{
+    "app-id": "org.flatpak.Formiko",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.34",
+    "sdk": "org.gnome.Sdk",
+    "command": "formiko",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=session-bus",
+        "--filesystem=home",
+        "--filesystem=/media",
+        "--filesystem=/keybase",
+        "--talk-name=org.gtk.vfs.*"
+    ],
+    "build-options" :
+    {
+        "cflags": "-O2 -g",
```

Comment:
Can be dropped, happen automatically with latest runtimes.

Reply:
dropped